core/kover/dataset/ds.py

```diff
@@ -26,7 +26,12 @@ class KoverDataset(object):
 	def __init__(self, file):
 		self.path = file
 		self.dataset_open = partial(_hdf5_open_no_chunk_cache, file)
-
+	
+	@property
+	def classification(self):
```

Comment:
I would use something like classification_type instead of classification (only for the name of the property, no need to change it in the file).